---
layout: page
title: Research
---

<p class="message">
Biology is carried out by interacting molecules. As fundamental as this concept is, we still do not know all molecular interactions in the cell. In particular, we do not know the identity of the molecular partners, we do not know the structure of their interactions, and we do not know how to control these interactions so we can modulate their function. Insufficient knowledge of these fundamentals is extremely limiting to our understanding of disease, our ability to develop therapeutics, and ultimately, our understanding of how life works. To gain this knowledge, we need tools. My lab builds tools to systematically characterize molecular interactions. Specifically, I develop tools to 1) identify all macromolecular assemblies in the cell, 2) characterize assemblies for factors that alter their stability (e.g. modifications, mutations, developmental stage, ligands), 3) determine the structure and function of these interactions, and 4) modulate interactions so we can control their function.
</p>

<p class="message">
	<h3>Identification of macromolecular assemblies</h3>
</p>
<p>
Complete understanding of the cell requires knowledge of the underlying molecular network (i.e. social network of biomolecules). How and when molecules interact ultimately determines a cell’s behavior. Unfortunately, we lack an accurate and comprehensive map of all biomolecular interactions. My lab builds machine learning and data mining tools to discover macromolecular assemblies in high throughput proteomic experiments. Towards this I have built the most accurate and comprehensive protein complex map available, which I call <a href="http://humap2.proteincomplexes.org" target='_blank'>hu.MAP</a>. Hu.MAP has been central to 1) identifying novel disease genes including developmental diseases such as ciliopathies, 2) functionally annotating completely uncharacterized genes, and 3) discovering altogether novel protein assemblies suggesting new uncovered cellular functions.
<br>
See:
<ul>
<li>
<a href="https://www.biorxiv.org/content/10.1101/2020.09.15.298216v1" target='_blank'>
Drew et al. hu.MAP 2.0: Integration of over 15,000 proteomic experiments builds a global compendium of human multiprotein assemblies. BioRxiv 2020
</a>
</li>
<li>
<a href="http://msb.embopress.org/content/13/6/932" target='_blank'>
Drew et al. Integration of over 9,000 mass spectrometry experiments builds a global map of human protein complexes. Molecular Systems Biology 2017
</a>
</li>
<li>
<a href="http://www.nature.com/nature/journal/v525/n7569/full/nature14877.html" target='_blank'>
Wan, Borgeson et al. Panorama of ancient metazoan macromolecular complexes. Nature 2015
</a>
</li>
<li>
<a href="https://www.sciencedirect.com/science/article/abs/pii/S0092867420302269" target='_blank'>
McWhite, Papoulas et al. A pan-plant protein complex map reveals deep conservation and novel assemblies. Cell 2020
</a>
</li>
</ul>
</p>
<!--
<img src="{{ site.url }}/images/complex_network.png" width="200" height="200" />
-->

<p class="message">
	<h3>Characterizing macromolecular assemblies</h3>
</p>
<p>
Although, we know the identity of many protein assemblies, we still know very little about what factors govern their different structural states. This lack of characterization severely inhibits our understanding of 1) how protein assemblies function, 2) how they assemble/disassemble, and 3) ultimately how they affect the global state of the cell. My lab develops high throughput proteomic methods to identify the cellular and biochemical factors (e.g. RNA/DNA/ligand content, post-translational modifications, genomic modifications, developmental stage) that govern macromolecular assemblies. Using these methods, we attempt to uncover general principles of assembly regulation and function.
</p>
<!--
<img src="{{ site.url }}/images/proteasome_correlation.png" height="200" />
<br>
(Wan, Borgeson et al. Nature 2015)
-->

<p class="message">
	<h3>Determining the structure of assemblies</h3>
</p>
<p>
A general principle of biomolecular function is that structure defines function. With knowledge of the 3D structure of biomolecules we can fully understand their mechanistic function and begin to investigate ways of controlling biomolecules for therapeutic purposes. Unfortunately, we lack 3D structure for thousands of protein assemblies. My lab develops methods to generate structural models for macromolecular assemblies providing testable hypotheses as to their mechanistic function.
</p>

<!--
<img src="{{ site.url }}/images/ja-2014-02310r_0005.jpeg" width="200" height="150" />
<br>
(Bullock Lao, Drew et al. JACS 2014)
-->

<p class="message">
	<h3>Modulation of molecular interactions</h3>
</p>
<p>
Improperly regulated protein interactions often cause disease but are difficult to target with traditional drug therapeutic approaches. My lab develops computational algorithms within the Rosetta Molecular Modeling Software Suite to design inhibitors that disrupt disease-causing interactions. My previous work in this area developed nanomolar binders to cancer drug targets (e.g. MDM2-P53 and HIF1α-P300). This work provides a foundation for modeling diverse non-protein based molecules in Rosetta, which greatly opens avenues to model a large swath of the chemical landscape such as peptidomimetics and oligosaccharides.
</p>

